packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -129,12 +130,50 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
     return super.validateCommand();
   }
 
+  Future<void> _validateXcodeBuildSettingsAfterArchive() async {
+
+    final BuildableIOSApp app = await buildableIOSApp;
+
+    final String plistPath = app.builtInfoPlistPathAfterArchive;
+
+    if (!globals.fs.file(plistPath).existsSync()) {
+      globals.printError('Invalid iOS archive. Does not contain Info.plist.');
+      return;
+    }
+
+    final Map<String, String?> xcodeProjectSettingsMap = <String, String?>{};
+    
```

Comment:
```
build_ios.dart:145: trailing U+0020 space character
```
```suggestion

```
Your IDE should have a "remove trailing spaces" editor option.
<img alt="Screen Shot 2022-10-24 at 11 55 47 AM" width="395" src="https://user-images.githubusercontent.com/682784/197603889-a52763ba-29f7-4ca3-bcd7-b6a55c557572.png">